libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -1460,10 +1394,9 @@ void AP_TECS::_update_throttle_limits() {
 
     // Configure min throttle.
 
-    // If less min throttle is allowed during takeoff, use it.
+    // Use takeoff min throttle, if applicable.
     bool use_takeoff_throttle = _flight_stage == AP_FixedWing::FlightStage::TAKEOFF || _flight_stage == AP_FixedWing::FlightStage::ABORT_LANDING;
-    const bool use_throttle_range = (aparm.takeoff_options & (uint32_t)AP_FixedWing::TakeoffOption::THROTTLE_RANGE);
-    use_takeoff_throttle = use_takeoff_throttle && (use_throttle_range == 1) && (aparm.takeoff_throttle_min != 0);
```

Comment:
`use_throttle_range` was a bug here on my part. When `TKOFF_OPTIONS=1` is irrelevant to using `TKOFF_THR_MIN`.

Granted, this section would be better if it was identical to how `TKOFF_THR_MAX` is used above.